python/sglang/srt/layers/moe/ep_moe/token_dispatcher.py

```diff
@@ -374,16 +377,22 @@ def combine(
             hidden_states, event, hook = self.combine_low_latency(
                 hidden_states, self.topk_idx, self.topk_weights, self.handle
             )
+
+        if self.async_finish:
+            event.current_stream_wait()
+
         self.handle = None
         return hidden_states.view(self.hidden_shape)
```

Comment:
`view` is redundant.

Reply:
Hmm that does not seem to be changed in this PR. Shall I change it here, or to create a separate PR (or maybe put in https://github.com/sgl-project/sglang/pull/4608 and rename title to "multiple tiny cleanups")?

![image](https://github.com/user-attachments/assets/07be70cc-9a65-401b-bb7c-45a267b347c0)